pkgs/development/python-modules/boltons/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pytest }:
+buildPythonPackage rec {
+  pname = "boltons";
+  version = "2018-12-31";
```

Comment:
I think the date of this commit is different https://github.com/mahmoud/boltons/commit/62fa806d9c86daf3f8c23e807cd853db2567b179

Also why not use the a release https://github.com/mahmoud/boltons/releases/tag/18.0.1?

Reply:
Release doesn't test :(  Date updated, and bumped the commit whilst I was at it.